src/Clp.jl

```diff
@@ -6,6 +6,7 @@ VERSION < v"0.7.0-beta2.199" && __precompile__()
 module Clp
 
 using Compat
+using Libdl
```

Comment:
Why is this needed ?

Reply:
Tu suppress a warning about Clp not having Libdl in its dependencies (Libdl is used by BinaryProvider), e.g. : https://github.com/bicycle1885/CodecZlib.jl/pull/26